README.adoc

```diff
@@ -26,78 +26,70 @@ This functions catalog is also a foundation for https://spring.io/projects/sprin
 |link:supplier/spring-ftp-supplier/README.adoc[FTP]
 |link:function/spring-header-enricher-function/README.adoc[Header-Enricher]
 |link:consumer/spring-elasticsearch-consumer/README.adoc[Elasticsearch]
-|
-|link:function/spring-header-filter-function/README.adoc[Header-Filter]
-|link:consumer/spring-file-consumer/README.adoc[File]
```

Comment:
I also removed the sparse spacing as the attempt to line up "like" supplier/function/consumer was a bit chaotic and was not lined up. Now they are just alpha ordered each column. 

<img width="626" alt="Screen Shot 2024-01-16 at 16 01 23" src="https://github.com/spring-cloud/spring-functions-catalog/assets/28907971/4c417d1f-d046-4ea5-9098-dcf3f78b733f">